plugins/guests/debian/cap/change_host_name.rb

```diff
@@ -18,7 +18,11 @@ def self.change_host_name(machine, name)
 
               # Prepend ourselves to /etc/hosts
               grep -w '#{name}' /etc/hosts || {
-                sed -i'' '1i 127.0.0.1\\t#{name}\\t#{basename}' /etc/hosts
+                if grep -w '^127\.0\.1\.1' /etc/hosts ; then
+                  sed -i'' 's/^127\.0\.1\.1\s/127.0.0.1\\t#{name}\\t#{basename}/' /etc/hosts
```

Comment:
I don't think this `sed` command actually works...I couldn't get it to work anyway. I think it should actually be:

```ruby
sed -i 's/^127\.0\.1\.1.*/127.0.1.1\\t#{name}\t#{basename}/g' /etc/hosts
```